src/connectors/subfire.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+$('#player').bind('playing pause timeupdate durationchange', Connector.onStateChanged);
```

Comment:
Why do you use the `playerSelector` property and add listener for these events at the same time? Is there a certain purpose for doing this?

Reply:
a real player is `#player-cover`. `#player` is just hidden `audio` tag in the bottom of the html to play the song.
when I try to make first connector version, I got a problem that this player receives the loading/updating events from the `#player`, updating progress and duration value would be delayed.

player process could be, starting queue -> update current track -> update `audio` tag -> playing + receiving duration change events.

so I want to get the similar timing of the original actions.
could i have an another option for it?

Reply:
I suppose adding an event listener for the `#player` element should be enough.

Assigning a value to the `playerSelector` property will make the extension to look for DOM changes of this element. I think it's redundant since we have the event listener set up. Or am I wrong?

Btw, how can I test the connector?

Reply:
I have been setuped [docker version of supysonic](https://github.com/glogiotatidis/supysonic-docker) into my own server
and I found some my mistakes (defined wrong method).
fixed review things; remove `$` and event handling of the `audio` tag.

and i also tested `playerSelector` to `audio#player`, it doesn't work, that this tag is always hidden and only changed the `src` value.